marketredesign/mrd-auth0-laravel
# Dataset authorization

**This is a public package, in a public repository**

## Proposed changes
Adds a middleware to authorize datasets. The underlying dataset repository is also exposed (using a facade) such that it can be used by applications using this package to retrieve the datasets a user has access to as well.

## Types of changes

What types of changes does your code introduce to this repository?


## Testing
Explain here how to run your unit tests, and explain how to execute manual testing. 

### Unit testing
Added.

### Manual testing
Use in laravel application, add dataset auth middleware (see readme) and verify that dataset access is authorized properly.

## Further comments
n/a
